mysql/src/main/scala/zio/sql/mysql/MysqlSqlModule.scala

```diff
@@ -27,12 +27,13 @@ trait MysqlSqlModule extends Sql { self =>
   }
 
   object MysqlFunctionDef {
-    val Crc32     = FunctionDef[String, Long](FunctionName("crc32"))
-    val Degrees   = FunctionDef[Double, Double](FunctionName("degrees"))
-    val Log2      = FunctionDef[Double, Double](FunctionName("log2"))
-    val Log10     = FunctionDef[Double, Double](FunctionName("log10"))
-    val Pi        = Expr.FunctionCall0[Double](FunctionDef[Any, Double](FunctionName("pi")))
-    val BitLength = FunctionDef[String, Int](FunctionName("bit_length"))
+    val Crc32       = FunctionDef[String, Long](FunctionName("crc32"))
+    val Degrees     = FunctionDef[Double, Double](FunctionName("degrees"))
+    val Log2        = FunctionDef[Double, Double](FunctionName("log2"))
+    val Log10       = FunctionDef[Double, Double](FunctionName("log10"))
+    val Pi          = Expr.FunctionCall0[Double](FunctionDef[Any, Double](FunctionName("pi")))
+    val BitLength   = FunctionDef[String, Int](FunctionName("bit_length"))
+    val CurrentDate = Expr.ParenlessFunctionCall0[LocalDate](FunctionName("current_date"))
```

Comment:
Could you please place the CurrentDate alphabetically after Crc32. And along the way BitLength at the begining?

Reply:
Sure. Thanks for the feedback!